docs/preview_conversion_analyzer.md

```diff
@@ -0,0 +1,93 @@
+# Converting To WinUI3 Using C# Conversion Analyzers
+
+## Background
+
+The purpose of the Microsoft.WinUI.Convert Nuget Package is to create a porting solution to allow Developers to convert WinUI2 projects to the new WinUI3 format.
+
+- WinUI is a native user experience (UX) framework for both Windows Desktop and UWP applications. WinUI ships as part of the Windows OS. 
+[More on WinUI](https://docs.microsoft.com/en-us/windows/apps/winui/)
```

Comment:
The WinUI website might be a better link to point to - http://aka.ms/winui (or also mention)

Reply:
Added link to WinUI website.